twitch.py

```diff
@@ -618,6 +617,10 @@ def __init__(self, settings: Settings):
         self._session: aiohttp.ClientSession | None = None
         self._auth_state: _AuthState = _AuthState(self)
         # GUI
+        if cli:
+            from cli import CLIManager as GUIManager
+        else:
+            from gui import GUIManager
```

Comment:
Because the entire project is typed and there would be now two classes with identical interfaces, one would have to refactor the entire thing into an abstract base class, and then subclass the two managers from it. Only then you can ensure the two interfaces match and are typed properly.